src/vs/workbench/parts/preferences/electron-browser/settingsEditor2.ts

```diff
@@ -47,9 +47,12 @@ import { CONTEXT_SETTINGS_EDITOR, CONTEXT_SETTINGS_SEARCH_FOCUS, CONTEXT_TOC_ROW
 import { IPreferencesService, ISearchResult, ISettingsEditorModel, ISettingsEditorOptions, SettingsEditorOptions, SettingValueType } from 'vs/workbench/services/preferences/common/preferences';
 import { SettingsEditor2Input } from 'vs/workbench/services/preferences/common/preferencesEditorInput';
 import { Settings2EditorModel } from 'vs/workbench/services/preferences/common/preferencesModels';
+import { IEditorGroupsService, IEditorGroup } from 'vs/workbench/services/group/common/editorGroupsService';
 
 const $ = DOM.$;
 
+const SETTINGS_URI = URI.parse('vscode://settingseditor');
```

Comment:
I'm not sure about this, editorMemento requires resource URI, but there is only one settings page. Probably there is a better way around this than hard-coded URI.